packages/client/src/modules/Account.ts

```diff
@@ -55,9 +55,11 @@ export class AccountModule {
      * For example, the USDC token id for Ethereum is `ethereum/erc20/usd__coin`.
      */
     currencyIds?: string[];
+    showAccountFilter?: boolean;
   }): Promise<Account> {
     const requestAccountsResult = await this.client.request("account.request", {
       currencyIds: params?.currencyIds,
+      showAccountFilter: params?.showAccountFilter ?? false,
```

Comment:
If it's optional we shouldn't need to default to false
```suggestion
      showAccountFilter: params?.showAccountFilter,
```